webrtc.html

```diff
@@ -5595,17 +5595,21 @@ <h2>Methods</h2>
                 has never been called, return a promise <a>rejected</a> with a newly
                 <a data-link-for="exception" data-lt="create">created</a>
                 <code>InvalidStateError</code>.</li>
-                <li>If <code><var>parameters</var>.encodings.length</code>
-                is different from <var>N</var>, or if
-                <code><var>parameters</var>.encodings</code> has been re-ordered,
-                or if any parameter in <var>parameters</var> is
-                marked as a <dfn>Read-only parameter</dfn> or has a value that is
-                different from the corresponding parameter value in <var>sender</var>'s
-                <a>[[\LastReturnedParameters]]</a> internal slot,
-                return a promise <a>rejected</a> with a newly
+                <li>If any of the following conditions are met, return a promise
+                <a>rejected</a> with a newly
                 <a data-link-for="exception" data-lt="create">created</a>
-                <code>InvalidModificationError</code>. Note that this also
-                applies to <var>transactionId</var>.</li>
+                <code>InvalidModificationError</code>:
+                <ol>  
+                   <li><code><var>parameters</var>.encodings.length</code>
+                   is different from <var>N</var>.</li>
+                   <li>Any parameter in <var>parameters</var> is
+                   marked as a <dfn>Read-only parameter</dfn> and has a value that is
+                   different from the corresponding parameter value in <var>sender</var>'s
+                   <a>[[\LastReturnedParameters]]</a> internal slot. Note that this also
+                   applies to <var>transactionId</var>.</li>
+                   <li><code><var>parameters</var>.encodings</code> has been re-ordered.</li>
```

Comment:
nit: May want to swap the second and third conditions, so that the ones dealing with `parameters.encodings` are next to each other.